apps/dsb-message-broker/src/message/message.query.pipe.ts

```diff
@@ -3,8 +3,24 @@ import { PipeTransform, Injectable, BadRequestException } from '@nestjs/common';
 @Injectable()
 export class MessageQueryPipe implements PipeTransform<any> {
     async transform(data: any) {
-        if (!data.fqcn) throw new BadRequestException('fqcn is missing in the query!');
-        if (typeof data.fqcn !== 'string') throw new BadRequestException('fqcn must be a string!');
+        if (data.from) {
```

Comment:
We need to add better validation for incoming parameters, also I suggest we create a DTO object with class-validator annotations as we use in other DTO classes, there are plenty of built in validator for date, regex etc

Reply:
sure. creating it.